### DISCUSSION TITLE
Widget iOS 16?

### DISCUSSION THREAD

**Discussion by hollow-frenk** [qnblackcat/uYouPlus#660] [Q&A] [answered]
Who can tell me why this app doesn’t support iOS widgets like the official app?

Strange, even the official app doesn’t have the widgets 👀 and yet they released them wtf?!

**Comment by qnblackcat** [comment, marked as answer]
Widget doesn‘t work when you sideload the app due to **many** reasons, and the wont be any fix. That’s all you need to know.
